src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String string;
+    private int variable;
```

Comment:
Please, consider more meaningful variable names here. It should be clear when reading code, for which purpose variable is created, like `age` or `size`.